tree/tree/src/TChain.cxx

```diff
@@ -1807,45 +1807,9 @@ void TChain::Lookup(bool force)
 ////////////////////////////////////////////////////////////////////////////////
 /// Loop on nentries of this chain starting at firstentry.  (NOT IMPLEMENTED)
 
-void TChain::Loop(Option_t* option, Long64_t nentries, Long64_t firstentry)
+void TChain::Loop(Option_t *, Long64_t, Long64_t)
```

Comment:
I think we can safely remove this routine altogether.  Its `TTree` sibling was removed 22 years ago: 5621165b06fd8a2bbffbe22009b9e66dd522d81b